charts/kong/values.yaml

```diff
@@ -505,7 +505,7 @@ enterprise:
   # Kong Enterprise license secret name
   # This secret must contain a single 'license' key, containing your base64-encoded license data
   # The license secret is required for all Kong Enterprise deployments
-  license_secret: you-must-create-a-kong-license-secret
+  license_secret: kong-enterprise-license
```

Comment:
Are changes in this file intentional?

Reply:
Yes. If we're using that name in the guide and examples, I don't see any reason the stock values.yaml should be any different.